index.html

```diff
@@ -139,5 +139,39 @@ <h2>Skills</h2>
           </ul>
     </section>
 </section>
+<!-------- Last Page -------->
+<section id="lastpage">
+  <nav>
+  </nav>
+  <header> 
+     <h1>Get started</h1>
+     <p id="paragraph">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+     <form>
```

Comment:
- [ ] Please @aliakhraisha 👋, make sure that your form includes an `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html).
     * `action` value should be similar to `https://formspree.io/f/{form_id}`
     * `method` value equals `POST`
  In your `form` you should have something like this 👇️

`<form action="https://formspree.io/f/{form_id}" method="post" >....</form>`

- [ ] First, go to [formspree](https://formspree.io/html) create an account and follow the instruction on how to set up an action.